uber/models.py

```diff
@@ -1247,6 +1395,27 @@ def _badge_adjustments(self):
     def _email_adjustment(self):
         self.email = normalize_email(self.email)
 
+    def _badge_adjustments(self):
```

Comment:
Huh, it looks like this is a mistake -- it's got the same name as a presave adjustment just above. It might be from merging. We should remove it.